### DISCUSSION TITLE
NextUI components in a Stitches project?

### DISCUSSION THREAD

**Discussion by jaukia** [heroui-inc/heroui#184] [Help] [answered]
Is it possible to use NextUI components directly in a Stitches project, without using the NextUIProvider? Would that work as long as the theme structure matches a NextUI theme, or is there something else that should be taken into account?

**Comment by jrgarciadev** [comment, marked as answer]
Hey @jaukia yes it is, just be careful because you won't have the `CssBaseline` included so some components may display in a different way

**Comment by jaukia** [comment]
Thanks, great! :)
